vpr/src/base/vpr_types.h

```diff
@@ -1015,9 +1015,11 @@ enum e_routing_failure_predictor {
     SAFE,
     AGGRESSIVE
 };
+
+// How to allocate budgets, and if RCV should be enabled
 enum e_routing_budgets_algorithm {
-    MINIMAX,
-    YOYO,
+    MINIMAX,          // Use MINIMAX-PERT algorithm to allocate budgets
+    YOYO,             // Use MINIMAX as above, and enable RCV algorithm to resolve negative hold slack
     SCALE_DELAY,
     DISABLE
```

Comment:
Should comment DISABLE too as it's an important one: currently turns off RCV and delay budgets and uses default router.